### PR TITLE
Move source code out of resources folder

### PR DESCRIPTION
**Pull request by patflynn**
@elharo @nbashirbello

### CODE REVIEW

**Comment by nbashirbello** [comment]
LGTM

File: core-plugin/src/com/google/gct/idea/appengine/run/AppEngineGradleBeforeRunTaskProvider.java [hunk 1]
```diff
@@ -29,15 +27,14 @@
 import com.intellij.openapi.module.Module;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.util.Key;
-import com.intellij.util.ThreeState;
-import icons.GoogleCloudToolsIcons;
+
+import com.google.gct.idea.ui.GoogleCloudToolsIcons;
```
Comment (elharo) [inline core-plugin/src/com/google/gct/idea/appengine/run/AppEngineGradleBeforeRunTaskProvider.java]: can you optimize the imports? these are out of order

Comment (patflynn) [inline core-plugin/src/com/google/gct/idea/appengine/run/AppEngineGradleBeforeRunTaskProvider.java]: done.

**Comment by elharo** [comment]
lgtm

**Pull request merged**